### ISSUE TITLE
error on Windows bash subsystem reading text

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#8324]
Hello,
I'm using Windows10 build 10.0.17672.1000
NVDA MASTER 15132,354ad355, that is 2018.2 :-)
Last development  version of Windows10 Essentials addon.

I installed the debian subsystem and in the console no problem.
if I want to use the nano editor, I obtain an error every time I press a key.
NVDA reads the characters, but something should be fixed... these are the last 5 6 lines of logs:

ERROR - NVDAObjects.behaviors.LiveText._monitor (09:45:16.673):
Error getting lines or calculating new text
Traceback (most recent call last):
  File "NVDAObjects\behaviors.pyc", line 278, in _monitor
  File "NVDAObjects\window\winConsole.pyc", line 49, in _getTextLines
  File "winConsoleHandler.pyc", line 125, in getConsoleVisibleLines
  File "wincon.pyc", line 56, in ReadConsoleOutputCharacter
WindowsError: [Error 16389] <no description>
